src/javascript/app/base/client.js

```diff
@@ -133,6 +133,20 @@ const Client = (() => {
         set('is_virtual', +authorize.is_virtual);
         set('session_start', parseInt(moment().valueOf() / 1000));
         set('landing_company_shortcode', authorize.landing_company_name);
+        updateAccountList(authorize.account_list);
+    };
+
+    const updateAccountList = (account_list) => {
+        account_list.forEach((account) => {
+            set('excluded_until', account.excluded_until, account.loginid);
```

Comment:
is this overwritten on line 146? do we need to explicitly mention it here?

Reply:
@negarn We need to mention it explicitly because if the client is not `excluded_until` no property is returned from backend for it.

Reply:
I see! should we change it to `account.excluded_until || ''` to be consistent with how we store undefined values on line 145 and in logged_in?